src/core/crash-handler.c

```diff
@@ -51,7 +52,7 @@ _noreturn_ static void crash(int sig, siginfo_t *siginfo, void *context) {
                 /* Pass this on immediately, if this is not PID 1 */
                 propagate_signal(sig, siginfo);
         else if (!arg_dump_core)
-                log_emergency("Caught <%s>, not dumping core.", signal_to_string(sig));
+                log_emergency(SD_MESSAGE_CRASH_NO_COREDUMP_STR, "Caught <%s>, not dumping core.", signal_to_string(sig));
```

Comment:
Good morning @bluca 

For instances where the existing calls to ```log_emergency*()``` have format strings like here,
how can I make the arguments fit the function definition of 2 args?

I'm thinking I should use ```asprintf()``` to get the message string with the signal first before passing to ```log_emergency*()```.